modules/images/dominant-color/class-wp-image-editor-gd-with-color.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @package performance-lab
+ * @group dominant-color
+ *
+ * @since 1.0.0
+ */
+
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @since 6.x
```

Comment:
Please use `n.e.x.t` per the Performance Lab plugin conventions, throughout this PR.
```suggestion
 * @since n.e.x.t
```